hudi-client/src/test/java/org/apache/hudi/index/TestHbaseIndex.java

```diff
@@ -168,13 +169,19 @@ public void testTagLocationAndDuplicateUpdate() throws Exception {
     HoodieWriteConfig config = getConfig();
     HBaseIndex index = new HBaseIndex(config);
     HoodieWriteClient writeClient = new HoodieWriteClient(jsc, config);
-    writeClient.startCommit();
+    writeClient.startCommitWithTime(newCommitTime);
     metaClient = HoodieTableMetaClient.reload(metaClient);
     HoodieTable hoodieTable = HoodieTable.getHoodieTable(metaClient, config, jsc);
 
     JavaRDD<WriteStatus> writeStatues = writeClient.upsert(writeRecords, newCommitTime);
     JavaRDD<HoodieRecord> javaRDD1 = index.tagLocation(writeRecords, jsc, hoodieTable);
+
     // Duplicate upsert and ensure correctness is maintained
+    // We are trying to approximately imitate the case when the RRD is recomputed. For RRD creating, driver code is not
```

Comment:
Did not follow this, lets chat f2f

Reply:
RDD :)

Reply:
Discussed this f2f and resolved my doubts